python/graphstorm/model/gnn.py

```diff
@@ -942,7 +942,8 @@ def do_full_graph_inference(model, data, batch_size=1024, fanout=None, edge_mask
     -------
     dict of th.Tensor : node embeddings.
     """
-    assert isinstance(model, GSgnnModel), "Only GSgnnModel supports full-graph inference."
+    assert isinstance(model, GSgnnModel) or type(model).__name__ == 'GLEM',\
```

Comment:
why do we need the assert? GLEM is a child class of `GSgnnNodeModelBase`

Reply:
GLEM is not a subclass of `GSgnnModel`.

Reply:
it is not ideal to explicitly check if it's GLEM. let me check which class we should assert.

Reply:
let's have it like this for now.